client/src/components/Artist/ArtistSupport.tsx

```diff
@@ -9,11 +10,27 @@ import { bp } from "../../constants";
 import FollowArtist from "components/common/FollowArtist";
 import SpaceBetweenDiv from "components/common/SpaceBetweenDiv";
 import { useArtistContext } from "state/ArtistContext";
-import { PostGrid } from "./ArtistPosts";
 import { useAuthContext } from "state/AuthContext";
 import { useQueryClient } from "@tanstack/react-query";
 import { QUERY_KEY_AUTH, queryKeyIncludes } from "queries/queryKeys";
 
+export const PostGrid = styled.div<{}>`
```

Comment:
Should this get imported from Post/PostGrid?

Reply:
Ah, I'd planned on renaming this to `SubscriptionGrid` since it's used for artist subscriptions and not posts. The changes to the `<PostGrid>` component also make reuse less intuitive here, IMO.